nilearn/group_sparse_covariance.py

```diff
@@ -9,6 +9,7 @@
 import collections
 import operator
 import itertools
+from past.builtins import xrange
```

Comment:
I see what happened; this was caused by rebase / merge / rebase confusion.  I eliminated the use of past completely now; none were actually meaningful.